docs/beginner/tutorial2-swapchain/README.md

```diff
@@ -197,7 +196,7 @@ event_loop.run(move |event, _, control_flow| {
         Event::WindowEvent {
             ref event,
             window_id,
-        } if window_id == window.id() => if !state.input(event) {
+        } if window_id == window.id() => if !state.input(event) { // <- This line changed after the "=>"
```

Comment:
Haha, I totally missed that this line had changed when I did the tutorial. I was contemplating on mentioning that since it made the "Challenge" in the end a bit more challenging 😄 

I also added it instead of the `_ => {}`, which made me wonder why it returned a bool at all 😄 

Basically, I think adding this comment is a very good idea 👍

Reply:
Cool